test/Microsoft.AspNet.Mvc.FunctionalTests/ModelBindingTest.cs

```diff
@@ -76,8 +97,8 @@ public class ModelBindingTest
 
             // Assert
             var stringValue = await response.Content.ReadAsStringAsync();
-            var isModelStateValid = JsonConvert.DeserializeObject<bool>(stringValue);
-            Assert.True(isModelStateValid);
+            var json = JsonConvert.DeserializeObject<ModelStateDictionary>(stringValue);
+            Assert.True(json.IsValid);
```

Comment:
nit: this isn't JSON, it's a `ModelStateDictionary` and should thus be named `modelState`